### ISSUE TITLE
Migrate device-db to rethinkdb

### ISSUE THREAD

**Issue by jonatasteixeira-zz** [openstf/stf-device-db#6]
Hello there!

I suggest using this project as a seed to populate a real table inside rethinkdb and then, refactoring the stf project to look at the database instead a npm module.

There are a lot of models of smartphones that are not included in stf-device-db.
If these data were in a table, it would be more easier to add new devices.

**Comment by sorccu** [comment]
Yeah we do have plans for something like that, but no time to actually do it unfortunately.

**Comment by jonatasteixeira-zz** [comment]
It's great to know that.

Could we keep this issue oppened just to follow up this question? 
We will try to figure out some ways to simplify the images updates. 

Thanks ;)

**Comment by sorccu** [comment]
Yes of course. And PRs are always welcome as well!
